config.yml

```diff
@@ -35,65 +35,90 @@ autorun_capability: false
 # Include extra tests: true or false
 include_extras: true
 
-badge_text: Community
+badge_text: mCODE EDITION
 
 # Resource validator options: must be one of "internal" or "external". external_resource_validator_url is only used if resource_validator is set to external.
-resource_validator: internal
-external_resource_validator_url: http://validator_service:4567
+resource_validator: external
+external_resource_validator_url: http://localhost:8081
```

Comment:
this should probably stay as `http://validator_service:45467` so that the docker version works out of the box.  Make sure to test that `docker-compose up` still works.  You may have to add the volume mount to the docker-compose file.